### ISSUE TITLE
Orca error when exporting big plots

### ISSUE THREAD

**Issue by mabreidi** [ropensci/plotly#1438]
Hello, 

I'm having trouble exporting fairly large plotly plots via orca(). Small plots, however, work just fine. 

For example...

```
testdata <- as.data.frame(matrix(rnorm(20000,30,5),10000,2))
p <- testdata %>% plot_ly(x=~V1,y=~V2,type="scatter",mode="markers")
orca(p,"test.png")
```

Returns this error:

>  Error in process_initialize(self, private, command, args, stdin, stdout,  : 
>   processx error, create process: #206 File name or extension too long
>  at 'win/processx.c:1039'

This however runs just fine:
```
smalltestdata <- as.data.frame(matrix(rnorm(1000,30,5),500,2))
psmall <- smalltestdata %>% plot_ly(x=~V1,y=~V2,type="scatter",mode="markers")
orca(psmall,"smalltest.png")
```
Does anyone know if and how this can be fixed? Or if there's another option to export my plotly plots (as svg and png)?

Thanks in advance!

**Comment by cpsievert** [comment]
Your 1st example runs ok for me. Please make sure you have the latest development version: `devtools::install_github("ropensci/plotly")`

**Comment by mabreidi** [comment]
Thanks, it's working now!

For anyone having the same problem:
I was pretty sure I had ithe latest version installed, but re-installed it. During installation I got an error saying that an older version of the httpuv package couldn't be removed. I then deleted the httpuv folder, reinstalled the development plotly  version. After the download library(plotly) gave me an error, saying some file was corrupt and plotly couldn't be loaded. When I closed RStudio and started it again, plotly loaded without any trouble and the orca() export worked just fine.
